doc/tools/makerst.py

```diff
@@ -106,6 +106,7 @@ def make_class_list(class_list, columns):
 
     f.close()
 
+
```

Comment:
duplicate newline that should be removed

Reply:
Now, I'm fine with removing it, though I do wanna mention that PEP8 states there should be two newlines between functions.

Reply:
Ah, okay, ignore me then :P